libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -319,6 +303,15 @@ private boolean startTelemetry() {
     return isTelemetryEnabled;
   }
 
+  boolean optLocationIn() {
```

Comment:
Why we changed the order of the methods? We normally organize the code in this particular order :point_right: `public`, `protected`, `private`, etc. trying to locate them in order of calling.